TACC/Core-CMS
# feat: add instagram, hide twitter

## Overview / Changes

- Add Instagram SVG.
- Add Instagram in social media template.
- Hide Twitter in social media template.

## Related

None.

## Testing

1. Open CMS instance.
2. Create snippet with "TEMPLATE:" set to `snippets/logos-social-media.html`.
3. Save template on page.
4. Verify you see social media icons: Facebook, YouTube, LinkedIn, Instagram.

## UI

| Create Snippet | Render Snippet |
| - | - |
| <img width="1070" alt="logos-social-media-2" src="https://github.com/TACC/Core-CMS/assets/62723358/72e5acae-0b53-46a0-8a95-e0bc39375250"> | ![logos-social-media-1](https://github.com/TACC/Core-CMS/assets/62723358/ba6efa16-1d69-4cba-946b-78ae794981d1) |

## Notes

Snippet renders giant icons, because there are no styles, because that is left to client unless CMD comes up with a standard pattern to display them. Currently, they are used only on https://www.tacc.utexas.edu/.
